database/models.py

```diff
@@ -1,56 +1,119 @@
-import sqlite3
-import matplotlib.pyplot as plt
+
+# coding: utf-8
+
+# In[1]:
+
+# Load the data
 from sqlalchemy.ext.declarative import declarative_base
 from sqlalchemy.orm import session
 from sqlalchemy import Column, Integer, String, DateTime, Float, create_engine
 from datetime import datetime
 
+
+# In[2]:
+
+# Create the main sqlalchemy class
 Base = declarative_base()
-class Train(Base):
-    __tablename__ = "train"
+
+
+# In[3]:
+
+# First Model corresponds to the incidences obtained in the database
+class Incidencia(Base):
+    __tablename__ = "incidences"
+
+    id = Column(Integer, primary_key=True)
+    tipo = Column(String, default="unknown")
+    autonomia = Column(String, default="unknown")
+    provincia = Column(String, default="unknown")
+    matricula = Column(String, default="unknown")
+    causa = Column(String, default="unknown")
+    fechahora_ini = Column(DateTime, default=None)
+    nivel = Column(String, default="unknown")
+    carretera = Column(String, default="unknown")
+    pk_inicial = Column(Float, default=1)
+    pk_final = Column(Float, default=1)
+    sentido = Column(String, default=1)
+    longitud = Column(Float, default=1.0)
+    latitud = Column(Float, default=1.0)
+
+
+# In[4]:
```

Comment:
This comments are part of jupyter-notebook when exporting files to python e.g.

![captura de pantalla de 2017-01-03 16-52-21](https://cloud.githubusercontent.com/assets/6491460/21613326/07dcf654-d1d5-11e6-9333-07b1173d713d.png)

I suggest that is not neccesary to remove or modify this comments. What do you think?